tests/README.md

```diff
@@ -13,6 +13,16 @@ We need all but the main `backend` service running.  Since multiple `backend` in
     # SETUP ALL IMAGES
     docker-compose up --build
 
+### Prepare your development environment
+
+Ensure pre-commit hooks are enabled. 
+
+**Do not enable on Windows** - `isort` is not given the correct parameters, it conflicts with travis verification, and will never pass.
```

Comment:
I faced this as well.
It has to do with the `.venv` being created within the topsrcdir. The code of `isort` to determine what is third party vs first is a recursive madness.